.github/pull_request_template.md

```diff
@@ -0,0 +1,47 @@
+**Description**
+
+<!-- 
+Please include a summary of the changes and which issue is fixed or which feature it added.
+Please also include relevant motivation and context. List any dependencies that are required for this change.
+-->
```

Comment:
What I meant by checkboxes was something like:
"This PR implements ...
- [x] feature_1
- [x] feature_2

including tests for `Component C` and does changes according to the adopted scheme ..."

This way, the reviewer has a quick and fast overview of what was implemented in the PR.

Reply:
I think this is still possible, as it would be included in the top, but this would limit the possibilities of the template, as this would only apply for PR implementing new features.

Reply:
This was just an example. I would just add the following to the template:
```
This PR implements...
- [ ] feature/ revision/ hotfix/ optimization/ ...

for/ of `Component`.
```
This list is not just helpful for the reviewer, but also for the assignees to check whether all features, revisions, etc. planned for this PR were implemented.